CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+CMAKE_MINIMUM_REQUIRED (VERSION 2.6)
+
+#add_definitions(-DOEM_TYPE=${OEM})
+
+#Required Packages
+PROJECT(leelaz)
+
+SET (BOOST_MIN_VERSION "1.58.0")
+FIND_PACKAGE(Boost ${BOOST_MIN_VERSION} REQUIRED)
+FIND_PACKAGE( BLAS REQUIRED )
+FIND_PACKAGE( ZLIB REQUIRED )
+FIND_PACKAGE( OpenCL REQUIRED )
+
+SET(IncludePath "CMAKE_CURRENT_SOURCE_DIR}/src")
+SET(SrcPath "${CMAKE_CURRENT_SOURCE_DIR}/src")
+
+INCLUDE_DIRECTORIES(${IncludePath})
+INCLUDE_DIRECTORIES(/usr/include/openblas)
+
+SET(SOURCES
+    "${SrcPath}/FastBoard.cpp"
+    "${SrcPath}/FastState.cpp"
+    "${SrcPath}/FullBoard.cpp"
+    "${SrcPath}/GameState.cpp"
+    "${SrcPath}/GTP.cpp"
+    "${SrcPath}/KoState.cpp"
+    "${SrcPath}/Leela.cpp"
+    "${SrcPath}/Network.cpp"
+    "${SrcPath}/OpenCL.cpp"
+    "${SrcPath}/Random.cpp"
+    "${SrcPath}/SGFParser.cpp"
+    "${SrcPath}/SGFTree.cpp"
+    "${SrcPath}/SMP.cpp"
+    "${SrcPath}/TimeControl.cpp"
+    "${SrcPath}/Timing.cpp"
+    "${SrcPath}/Training.cpp"
+    "${SrcPath}/TTable.cpp"
+    "${SrcPath}/UCTNode.cpp"
+    "${SrcPath}/UCTSearch.cpp"
+    "${SrcPath}/Utils.cpp"
+    "${SrcPath}/Zobrist.cpp"
+)
+
+ADD_EXECUTABLE(leelaz ${SOURCES})
+
+TARGET_LINK_LIBRARIES(leelaz openblas)
```

Comment:
Similar remark as above, shouldn't this use the output from the FIND_PACKAGE thing? What if the user has ATLAS or another BLAS lib?